cmd/main.go

```diff
@@ -385,12 +380,39 @@ func createSkrWebhookManager(mgr ctrl.Manager, skrContextFactory remote.SkrConte
 	if err != nil {
 		return nil, err
 	}
+
+	certificateConfig := certificate.CertificateConfig{
+		Duration:    flagVar.SelfSignedCertDuration,
+		RenewBefore: flagVar.SelfSignedCertRenewBefore,
+		KeySize:     flagVar.SelfSignedCertKeySize,
+	}
+
+	certificateManagerConfig := certificate.CertificateManagerConfig{
+		SkrServiceName:               watcher.SkrResourceName,
+		SkrNamespace:                 flagVar.RemoteSyncNamespace,
+		CertificateNamespace:         flagVar.IstioNamespace,
+		AdditionalDNSNames:           strings.Split(flagVar.AdditionalDNSNames, ","),
+		GatewaySecretName:            shared.GatewaySecretName,
+		RenewBuffer:                  flagVar.SelfSignedCertRenewBuffer,
+		SkrCertificateNamingTemplate: flagVar.SelfSignedCertificateNamingTemplate,
+	}
+
+	certificateManager := certificate.NewCertificateManager(
```

Comment:
based on https://github.com/kyma-project/lifecycle-manager/pull/2382/files#r2030696256, the initiation of `CertificateManager` could be inside `SKRWebhookManifestManager`, since it's an internal component.

Reply:
See above comment on the import.

Reply:
We have decided to move the discussion and decision to https://github.com/kyma-project/lifecycle-manager/pull/2393 and https://github.com/kyma-project/lifecycle-manager/pull/2395. We may refactor this again depending on the decision taken there.